### ISSUE TITLE
do file method fails in debugger

### ISSUE THREAD

**Issue by beltaurus** [richterger/Perl-LanguageServer#51]
Consider the simple: `main.pl`:
```
package main 1.0;
use 5.010;
use strict;
use warnings;
use Carp;
use English '-no_match_vars';

do './File/Test.pl' or croak "$OS_ERROR\n$EVAL_ERROR";
1;
```
Even if that do-file exists:
```
print "Loading File/Test.pl\n";
```
invoking `perl main.pl` prints the message, no error messages.

But launching the debugger in VS Code fails with this error:
```
Could not load source './File/Test.pl': Unknow perlmethod _dapreq_source at /mnt/h/dseynhae/perl5/lib/perl5/Perl/LanguageServer.pm line 217.
```

**Comment by beltaurus** [comment]
If I provide the full path to the do-file, things work...

**Comment by richterger** [comment]
I guess under the debugger your program was run in a different path, so Test.pl was not found. Use cwd option to spceify the correct path for program execution (after I have fixed #43, fix is comeing soon)
